src/lib/web-component/log-view/log-view-element.js

```diff
@@ -71,9 +73,6 @@ export class LogView extends HTMLElement {
       if (log.type === 'content_script') {
         const contentScriptTD = logTableRowInstance.querySelector('.name');
         contentScriptTD.textContent = log.name;
-        contentScriptTD.colSpan = 2;
```

Comment:
Since we're no longer merging the columns, the whole logic here can be simplified by removing the `log.type` comparison. For example to

```javascript
      logTableRowInstance.querySelector('.api-name').textContent = log.name;
      logTableRowInstance.querySelector('.view-type').textContent = log.viewType || "";
```

Reply:
Thanks. I have updated this section.